docker-compose.yml

```diff
@@ -137,6 +137,30 @@ services:
       - "logging=true"
     restart: always
     logging: *default-logging
+  login-dashboard:
+    image: lblod/acmidm-login-service:0.11.0
+    environment:
+      MU_APPLICATION_AUTH_DISCOVERY_URL: "https://authenticatie-ti.vlaanderen.be/op"
+      MU_APPLICATION_AUTH_CLIENT_ID: "a2c0d6ea-01b4-4f68-920b-10834a943c27"
+      MU_APPLICATION_AUTH_REDIRECT_URI: "https://dashboard.lblod.info/authorization/callback"
+      MU_APPLICATION_AUTH_CLIENT_SECRET: "secret"
+      LOG_SINK_URL: "http://sink"
+    labels:
+      - "logging=true"
+    restart: always
+    logging: *default-logging
+  login-vendor-management:
```

Comment:
Did you add this in preparation for the ACM/IDM setup in the dashboard or something? :smile:

Reply:
For the vendor-management, yes. Not good? One PR for everything?